Assets/HoloToolkit/UI/Textures/MRTK_Logo.png.meta

```diff
@@ -0,0 +1,94 @@
+fileFormatVersion: 2
+guid: f721996453d888a4db83f0f9f1a4eb7c
```

Comment:
Images should be a power of 2, so mipmapping works better (eg, 64, 128, 512, 1024, 2048, etc)

Reply:
@StephenHodgson Updated the image file to 2048x512px. Thanks!